internal/commands/prepare/prepare.go

```diff
@@ -181,6 +181,43 @@ func runPrepareVM(cmd *cobra.Command, args []string) error {
 		log.Printf("Timezone was set to %s!\n", tz)
 	}
 
+	dirsToMount := []string{}
+	if config.HostDir {
+		dirsToMount = append(dirsToMount, "hostdir")
+	}
+	if _, ok := os.LookupEnv(tart.EnvTartExecutorInternalBuildsDir); ok {
+		dirsToMount = append(dirsToMount, "buildsdir")
+	}
+	if _, ok := os.LookupEnv(tart.EnvTartExecutorInternalCacheDir); ok {
+		dirsToMount = append(dirsToMount, "cachedir")
+	}
+
+	for _, dirToMount := range dirsToMount {
+		log.Printf("Mounting %s...\n", dirToMount)
+
+		session, err := ssh.NewSession()
+		if err != nil {
+			return err
+		}
+		defer session.Close()
+
+		mountScript := fmt.Sprintf(
+			"mount_virtiofs tart.virtiofs.%s /Users/%s/%s\n",
```

Comment:
`mount_virtiofs ` expects a directory to be already created, so we should probably stuff an `mkdir -p && ...` here to avoid the job failing.

Reply:
Great catch. See [012a73f](https://github.com/cirruslabs/gitlab-tart-executor/pull/65/commits/012a73f1532e27a395a41afb3d4b9747c4e83f6c)